turbinia/state_manager.py

```diff
@@ -31,20 +32,22 @@
   pass
 
 from google.cloud import datastore
+from google.cloud import exceptions
 
 from turbinia import config
 from turbinia import TurbiniaException
 from turbinia.workers import TurbiniaTask
 from turbinia.workers import TurbiniaTaskResult
 
 DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S'
+MAX_DATASTORE_STRLEN = 1500
```

Comment:
This is only for indexed strings, unindexed strings have a limit of 1MB. Does this property need to be indexed?

Ref: https://cloud.google.com/datastore/docs/concepts/limits

Reply:
So it looks like we don't actually index this field, and I'm not actually sure why it's giving these errors.  I filed #217 to figure this out.   In the meantime if possible I'd like to submit this because it fixes a few issues, and we can easily raise the limits once we figure out what's going on. Thanks!